Glideslope.json

```diff
@@ -5,7 +5,6 @@
    "identifier": "IWG",
    "associatedRunwayNumber": "14L",
    "frequency": 110100,
-   "elevation": 129,
```

Comment:
Somehow this line has been removed, you'll need to re-add it. That's why the build is failing.